lib/datastore/updater.es6.js

```diff
@@ -0,0 +1,83 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+require('../confluence/set_ops.es6.js');
+
+foam.CLASS({
+  package: 'org.chromium.apis.web',
+  name: 'DatastoreUpdater',
+  implements: ['org.chromium.mlang.Expressions'],
+
+  imports: ['info'],
+
+  methods: [
+    function unversionData(versionedDAO, unversionedDAO) {
+      return versionedDAO.select().then(sink => {
+        const array = sink.array;
+        this.info(`Unversioning ${array.length} ${versionedDAO.of.id} objects`);
+        const cls = unversionedDAO.of;
+        let promises = [];
```

Comment:
Heh, this is exactly like some code I wrote just yesterday. I used const here though, but OK to use let if you have a convention about mutability that uses let even when const would work.

Reply:
Yeah. I tend to use let if I *know* I intend to mutate *something* within the object (even though that's not what ES2015 enforces :roll_eyes:)

Reply:
That's fine, I'll know to expect just that going forward.